docker/scripts/mariadb-init.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Why switching to a bash script?
The sql file is the way proposed in the mysql container doc.

Reply:
Using a bash script is an option too (see https://hub.docker.com/_/mariadb).
Doing this, we can change database user based on environement vars. Using your sql, we need to set user to "squest_user"

Reply:
All good then.